### PR TITLE
fix variable names in CE Losses

### LINKED ISSUE #1138: Some confusion in CrossEntropyLoss
As the title, there is some confusion in [CrossEntropyLoss](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/cross_entropy_loss.py). For example, the ``weight`` used in ``forward()`` ([L#95](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L89)) is the ``label`` for ``mask_cross_entropy`` ([#L54](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L54)). The variable names seems to not reflect their roles. Some suggestions for improvement could be:
1. [#L87](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L87): change ``cls_score`` to ``pred``.
2. [#L54](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L54): change ``target`` to ``label``.
3. [#L54](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L54): change ``label`` to ``mask``, add argument ``weight`` for future use. It would be make ``mask_cross_entropy`` consistent with other CE losses

### PR DESCRIPTION
**Pull request by thangvubk**
Fix #1138

### CODE REVIEW

**Comment by hellock** [comment]
I ran the new version for twice but it seems that the mAP of Mask R-CNN is 0.1~0.2 lower than the original version. That's weird.

**Comment by thangvubk** [comment]
Is the mAP of original Mask R-CNN also re-trained or from modelzoo?

**Comment by thangvubk** [comment]
@hellock  I just retrain the latest version without this modification and the results are also lower than model zoo 0.1 AP: 
``BboxAP/MaskAP = 37.2/34.1``
It seems that some harmful intermediate commits have made the performance changes.

**Comment by hellock** [comment]
I rerun some baselines and found that the results are not very stable. Sometimes the mAP can be 0.1 or 0.2 lower, and sometimes just the same. Basically the latest version is slightly lower than the results in the model zoo.
